### ISSUE TITLE
One city different name

### ISSUE THREAD

**Issue by paingheinthu** [outlandnish/poc-vpn-detection#1]
`Asia/Rangoon` is the same `Asia/Yangon`. Using different spelling for one city

**Comment by outlandnish** [comment]
Yeah, it looks like that's an issue with the timezones that's the ip -> geolocation service is returning. Thanks for the heads up!
